stores/metadata_test.go

```diff
@@ -280,7 +280,7 @@ func TestSQLContractStore(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if len(sets) != 2 {
+	if len(sets) != 3 { // 2 sets + default set
```

Comment:
`nit` perhaps assert value of `sets[2]` in the assertion below while we're at it?